benchmark/arm-manipulation-cg.cpp

```diff
@@ -8,9 +8,9 @@
 
 #ifdef WITH_MULTITHREADING
 #include <omp.h>
-#define NTHREAD 4
+#define NUM_THREADS WITH_NTHREADS
```

Comment:
Where do you define WITH_NTHREADS?

Reply:
It is a cmake flag. When you enable the multhreading then it asks you for this.

Reply:
I don't see WITH_NTHREADS anywhere. What you are defining is named BUILD_WITH_NTHREADS.

Reply:
I double checked it and tracked the issue.
For some reason, you removed the cmake lines that were passing this variable (in a previous PR). I will push the code with the fix in a second.

Reply:
I pushed the changes. I marked this conversation as resolved. Feel free to change its status if you need to keep discussing.